doc/rtd/reference/breaking_changes.rst

```diff
@@ -11,6 +11,45 @@ releases.
     many operating system vendors patch out breaking changes in
     cloud-init to ensure consistent behavior on their platform.
 
+24.3
+====
+
+Single Process Optimization
+---------------------------
+
+As a performance optimization, cloud-init no longer runs as four seperate
+Python interpretters. Instead, it launches a single process and then
```

Comment:
```suggestion
Python invocations. Instead, it launches a single process and then
```

`interpreters` makes me think we have independent interpreters installed or something.